modules/Link.js

```diff
@@ -73,6 +74,11 @@ const Link = React.createClass({
   },
 
   handleClick(event) {
+    invariant(
```

Comment:
I would move this to a lifecycle hook (probably `componentWillMount`) so developers can be alerted to mis-use earlier. I could see a Link not getting clicked on in testing and then going to production broken.